README.md

```diff
@@ -74,30 +74,25 @@ In order for Chart.js to obey the custom size you need to set `maintainAspectRat
 
 ```js
 <Bar
-	data={data}
-	width={100}
-	height={50}
-	options={{
-		maintainAspectRatio: false
-	}}
+  data={data}
+  width={100}
+  height={50}
+  options={{ maintainAspectRatio: false }}
 />
 ```
 
 ### Chart.js instance
 Chart.js instance can be accessed by placing a ref to the element as:
 
 ```js
-
 chartReference = {};
 
 componentDidMount() {
-	console.log(this.chartReference); // returns a Chart.js instance reference
+  console.log(this.chartReference); // returns a Chart.js instance reference
 }
 
 render() {
-	return (
-		<Doughnut ref={(reference) => this.chartReference = reference } data={data} />
-	)
+  return (<Doughnut ref={(reference) => this.chartReference = reference } data={data} />)
```

Comment:
perhaps you can shorten it a bit to this:

```
render() {
  return <Doughnut ref={chart => this.chartReference = chart} data={data} />;
}
```

or use the new `React.createRef()` API, maybe.